src/pages/Home/index.js

```diff
@@ -387,7 +479,7 @@ const Home = () => {
             onUpload={onImageUpload}
             text={
               selectedSize &&
-              uploadedImages[selectedSize.size][selectedSize.index]
+                uploadedImages[selectedSize.size][selectedSize.index]
```

Comment:
Not sure why the need for this change? Was it an automatic change made by prettier?

Reply:
I ran 'Format Document' in vscode, it may not be using the prettier config.

Reply:
I'll tidy up.

Reply:
See below: `yarn run pretty-quick` will fix everything automatically for you.